content/en/logs/guide/access-your-log-data-programmatically.md

```diff
@@ -0,0 +1,524 @@
+---
+title: Programmatically Access Log Data Using the Logs Search API
+kind: guide
+further_reading:
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Learn more about Log Explorer"
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Learn more about Search Syntax"
+- link: "/logs/search_syntax/"
+  tag: "Documentation"
+  text: "Learn more about syntax for Logs Search API"
+---
+
+
+## Overview
+
+Use the [Logs Search API][1] to programmatically access your log data and execute queries.
+
+The following examples are covered in this guide:
+
+* [Basic search](#basic-search)
+* [Sort by facet or timestamp](#sort-by-facet-or-timestamp)
+* [Limit the number of results retrieved](#limit-the-number-of-results-retrieved)
+* [Pagination](#pagination)
+
+## Prerequisites
+
+- An API key and an application key from an admin user is required to use the API. These are available in your [Datadog account API key page][2]. Replace `<DATADOG_API_KEY>` and `<DATADOG_APP_KEY>` with your Datadog API key and your Datadog application key.
+
+- This guide features `curl` examples. Install [curl][3] if you do not have it installed, or reference additional language examples for this API endpoint in the [API documentation][1].
+
+## Examples
+
+### Basic search
+
+To retrieve all log events within a specific time period, use the following [Search Syntax][4] to complete the API call.
+
+`from` indicates the `start time` and `to` indicates the `end time` for the log data. `query` indicates the search query that must be executed.
+
+**API call:**
+
+```bash
+
+curl -L -X POST "https://api.{{< region-param key="dd_site" code="true" >}}/api/v2/logs/events/search" -H "Content-Type: application/json" -H "DD-API-KEY: <DATADOG_API_KEY>" -H "DD-APPLICATION-KEY: <DATADOG_APP_KEY>" --data-raw '{
+  "filter": {
+    "from": "2020-10-07T00:00:00+00:00",
+    "to": "2020-10-07T00:15:00+00:00",
+    "query": "*"
+  }
+}'
+
+```
+
+**Response:**
+
+The result dataset is comprised of the `data` object, as depicted in the following example response.
+
+```json
+
+{
+    "meta": {
+        "page": {
+            "after": "eyJhZnRlciI6IkFRQUFBWFVBWFJZVkpHeHZEUUFBQUFCQldGVkJXRkpaVm1kMlpsa3RiVWRVWmpSQlFRIn0"
+        }
+    },
+    "data": [
+        {
+            "attributes": {
+                "status": "info",
+                "service": "pageViewService",
+                "tags": [
+                    "source:postman",
```

Comment:
I know it's only dummy content, but could we use some real world example here. 
Something on that model for instance: https://datadog-docs-staging.imgix.net/images/logs/guide/sensitive/sensitive_outline_query.29475f4f0e37d1455a4cb18306fcd278.png?fit=max&auto=format